File: Assignment14/Question4.c
# include <stdio.h>

void Pattern(int iRow, int iCol) 
{
  int i = 0;
  for (i = 1; i <= iRow; i++) 
  {
    int j = 0;
    for (j = 1; j < iCol; j++) 
    {
      if ((i % 2) == 1) 
      {
        printf("%d\t", j);
      }
      else 
      {
        printf("%d\t", -j);
      }
    }
    printf("\n");
  }
}

int main() 
{
  int iValue1 = 0;
  int iValue2 = 0;

  printf("Enter the number of rows : ");
  scanf("%d", &iValue1);

  printf("Enter the number of Columns : ");
  scanf("%d", &iValue2);

  Pattern(iValue1, iValue2);

  return 0;
}

File: Assignment50/Question4.java
import java.util.Scanner;

class Pattern 
{
    public void Pattern(int iRow, int iCol) 
    {
        int i = 0 , j = 0 ; 
        if (iRow <= 0 || iCol <= 0) 
        {
            return;
        }

        for (i = 1; i <= iRow; i++) 
        {
            for (j = 1; j <= iCol; j++) 
            {
                
                if (i == 1 || i == iRow || j == 1 || j == iCol) 
                {
                    System.out.print("*\t");
                }
                
                else if (j < (iCol - i + 1)) 
                {
                    System.out.print("#\t");
                }
                
                else if (j > (iCol - i + 1)) 
                {
                    System.out.print("$\t");
                }
                
                else 
                {
                    System.out.print("*\t");
                }
            }
            System.out.println();
        }
    }
}

public class Question4 
{
    public static void main(String A[]) 
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number of rows: ");
        int iValue1 = sobj.nextInt();
        
        System.out.println("Enter number of columns: ");
        int iValue2 = sobj.nextInt();
        

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1,iValue2);
    }
}